pkg/manager/options.go

```diff
@@ -138,3 +136,9 @@ func (o *Options) getCredentials() map[string]string {
 
 	return credentials
 }
+
+func (o *Options) updateCredentials() {
```

Comment:
nit: more appropriate name for the function
```suggestion
func (o *Options) readAndSetCredentials() {
```